hack/install.sh

```diff
@@ -29,6 +29,48 @@ err() {
   exit 1
 }
 
+# Check if the terminal supports ANSI escape codes
+if [[ $TERM == *256color* || $TERM == *xterm* ]]; then
+  RESET='\033[0m'
+  BOLD='\033[1m'
+  BLUE_BG='\033[44m'  # Blue background
+  BLUE_FG='\033[34m'  # Blue foreground
+else
+  RESET=''
+  BOLD=''
+  BLUE_BG=''  # No color support
+  BLUE_FG=''
+fi
+
+# Function to draw the animated progress bar with a blue background and bold text
+draw_animated_bar() {
```

Comment:
![Screenshot 2024-08-29 at 08 55 08](https://github.com/user-attachments/assets/38c67a6f-10be-48ec-923f-7196f9762913)

I get a lot of errors when running this.